data-prepper-plugins/otel-proto-common/src/main/java/org/opensearch/dataprepper/plugins/otel/codec/OTelLogsInputCodecConfig.java

```diff
@@ -19,12 +19,18 @@
         "It creates a Data Prepper log event for each log record along with the resource attributes in the file.")
 public class OTelLogsInputCodecConfig {
     static final OTelLogsFormatOption DEFAULT_FORMAT = OTelLogsFormatOption.JSON;
+    static final OTelOutputFormat DEFAULT_OTEL_FORMAT = OTelOutputFormat.OPENSEARCH;
     
     @JsonProperty(value = "format", defaultValue = "json")
     @JsonPropertyDescription("Specifies the format of the OTel logs.")
     @NotNull
     private OTelLogsFormatOption format = DEFAULT_FORMAT;
  
+    @JsonProperty(value = "otel_format", defaultValue = "opensearch")
+    @JsonPropertyDescription("Specifies the format of the OTel Output.")
+    @NotNull
+    private OTelOutputFormat otelFormat = OTelOutputFormat.OPENSEARCH;
```

Comment:
This makes sense.

But, why do you change the enum above? See my other comment.